File: FootScout-Vue.WebAPI/Models/DTOs/ClubHistoryCreateDTO.cs
using FootScout_Vue.WebAPI.Entities;

namespace FootScout_Vue.WebAPI.Models.DTOs
{
    public class ClubHistoryCreateDTO
    {
        public int PlayerPositionId { get; set; }
        public string ClubName { get; set; }
        public string League { get; set; }
        public string Region { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public Achievements Achievements { get; set; }
        public string PlayerId { get; set; }
    }
}

File: FootScout-Vue.WebClient/src/models/dtos/ClubHistoryCreateDTO.ts
import type { Achievements } from './AchievementsDTO';

// Reprezentuje historię klubu zawodnika, zawierając informacje o pozycji, nazwie klubu, lidze, 
// regionie, datach oraz osiągnięciach.

export interface ClubHistoryCreateDTO {
    playerPositionId: number;
    clubName: string;
    league: string;
    region: string;
    startDate: string;
    endDate: string;
    achievements: Achievements;
    playerId: string;
}